[4.4] docs(textarea):Add cols and rows parameters in the overview (#1527)
telerik/blazor-docs: components/textarea/overview.md

| ----------- | ----------- | ----------- |
| `Value` | `string` | Get/set the value of the input, can be used for binding. |
| `AutoComplete` | `bool` | Maps to the autocomplete attribute of the HTML `<textarea>`. |
| `AutoSize` | `bool` | Specifies if the TextArea will adjust its height based on the user input. |
| `Class` | `string` | The custom CSS class to be rendered on the `<span class="k-textarea">` element. |

> | `Cols` | `int?` | Maps to the `cols` attribute of the HTML `<textarea>` element. Not supported in combination with `Width`.

Reviewer: Also consider "Do not use together with `Width`."

```suggestion
| `Cols` | `int?` | Maps to the `cols` attribute of the HTML `<textarea>` element. Not supported in combination with `Width`.
```

> | `Cols` | `int?` | Maps to the `cols` attribute of the HTML `<textarea>` element. Not supported in combination with `Width`.

Author: ```suggestion
| `Cols` | `int?` | Maps to the `cols` attribute of the HTML `<textarea>` element. Do not use together with `Width`.
```

| `DebounceDelay` | `int` | Specifies the time in milliseconds between the last typed symbol and the updating of the value. The default value is 150ms. |
| `Enabled` | `bool` <br /> `true` | Whether the TextArea is enabled. |
| `Id` | `string` | Renders as the `id` attribute on the `<textarea>` element, so you can attach a `<label for="">`. |
| `Name` | `string` | The `name` attribute of the HTML element. It is usually required so the `AutoComplete` will be honored by the browser. |
| `Placeholder` | `string` | A `string` that maps to the `placeholder` attribute of the HTML element. If a `Label` is defined, it will be shown instead of the placeholder when the input is not focused. |

> | `Rows` | `int?` | Maps to the `rows` attribute of the HTML `<textarea>` element.

Reviewer: Is it supported together with `Height` ?
